crates/sui-sdk/src/lib.rs

```diff
@@ -36,7 +36,7 @@
 //! the Sui devnet, and the Sui testnet is shown below.
 //! To successfully run this program, make sure to spin up a local
 //! network with a local validator, a fullnode, and a faucet server
-//! (see [here](https://github.com/stefan-mysten/sui/tree/rust_sdk_api_examples/crates/sui-sdk/examples#preqrequisites) for more information).
+//! (see [https://github.com/MystenLabs/sui/tree/main/crates/sui-sdk/examples) for more information).
```

Comment:
```suggestion
//! (see the [Rust SDK examples](https://github.com/MystenLabs/sui/tree/main/crates/sui-sdk/examples) on GitHub for more information).
```
Gives the user an idea of where they're headed before they click.